tests/unit/backend/corpora/dataset_processing/test_process_cxg.py

```diff
@@ -0,0 +1,53 @@
+import pathlib
+import tempfile
+from unittest.mock import patch
+
+from backend.corpora.common.corpora_orm import (
+    DatasetArtifactFileType,
+)
+
+from backend.corpora.common.entities.dataset import Dataset
```

Comment:
Moved test here from test_process.py. The only changes are in the imports and some of the mock.patches